packages/react/src/components/Notification/Notification.js

```diff
@@ -471,6 +497,11 @@ export class InlineNotification extends Component {
             {this.props.children}
           </NotificationTextDetails>
         </div>
+        {actions.primary &&
```

Comment:
I meant just `{actions}` here, so user of notification can do `<InlineNotification ... actions={<NotificationActionButton>Action button</NotificationActionButton>}>`